apple-app-site-association.json

```diff
@@ -25,8 +25,6 @@
           "NOT /2016-year-in-art",
           "NOT /venice-biennale*",
           "NOT /gender-equality*",
-          "NOT /collections",
```

Comment:
Did we build this index view in the app? I _think_ we'll still want users to see the web version of this (but could totally be missing something).

Reply:
+1 I think we still want to keep this line for now.

Reply:
Good catch. I just corrected it.